examples/amm/src/contract.rs

```diff
@@ -217,6 +312,15 @@ impl Amm {
         result: &mut ExecutionResult<Message>,
         operation: OperationType,
     ) -> Result<(), AmmError> {
+        if let OperationType::RemoveLiquidity {
+            owner: _,
+            input_token_idx: _,
+            input_amount: _,
+        } = operation
+        {
+            return Err(AmmError::RemovingLiquidityFromRemoteChain);
+        }
```

Comment:
But we still allow adding liquidity?
I guess it's fine; it's basically a donation. But a user might be doing it by mistake, assuming they'd get shares or something.

Reply:
I'll just remove it as well then, since we'll eventually give shares for that. So it's not too confusing